src/main/java/com/google/firebase/remoteconfig/TagColor.java

```diff
@@ -0,0 +1,26 @@
+package com.google.firebase.remoteconfig;
+
+public enum TagColor {
+  BLUE("BLUE"),
+  BROWN("BROWN"),
+  CYAN("CYAN"),
+  DEEP_ORANGE("DEEP_ORANGE"),
+  GREEN("GREEN"),
+  INDIGO("INDIGO"),
+  LIME("LIME"),
+  ORANGE("ORANGE"),
+  PINK("PINK"),
+  PURPLE("PURPLE"),
+  TEAL("TEAL"),
+  UNSPECIFIED("CONDITION_DISPLAY_COLOR_UNSPECIFIED");
+
+  private final String color;
+
+  TagColor(String color) {
```

Comment:
Can this be private?

Reply:
It looks like that the Enum constructor is implicitly private. So we should be good here.